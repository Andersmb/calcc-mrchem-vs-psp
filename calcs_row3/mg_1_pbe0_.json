{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -0.13015525627515415
          ],
          "radius": 1.72
        },
        {
          "center": [
            0.0,
            0.0,
            3.138503783724846
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "mg",
          "xyz": [
            0.0,
            0.0,
            -0.13015525627515415
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            3.138503783724846
          ]
        }
      ],
      "multiplicity": 2
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "mg_1_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        5.605775527623769e-18
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.5052624300165808,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -1.5802216565242517e-07,
            -3.7910976844403175e-08,
            -0.5052624300165547
          ],
          "vector_el": [
            -1.5809459502887186e-07,
            -3.7983315527981717e-08,
            -2.0819031383848494
          ],
          "vector_nuc": [
            7.242937644667794e-11,
            7.233868357854368e-11,
            1.5766407083682947
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Mg",
          "xyz": [
            0.0,
            0.0,
            -0.13015525627515415
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            3.138503783724846
          ]
        }
      ],
      "multiplicity": 2,
      "orbital_energies": {
        "energy": [
          -46.92530929790521,
          -3.1612471976777123,
          -1.8842484608294787,
          -1.8842484598996143,
          -1.880671865341123,
          -0.30295994966737677,
          -0.18106757607028595,
          -46.92329798769589,
          -3.1573905925081083,
          -1.8818333659847593,
          -1.8818333624868455,
          -1.875351411928238,
          -0.27736671870787155
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -112.21682624670251
      },
      "scf_energy": {
        "E_ee": 99.72782462847287,
        "E_eext": 0.0,
        "E_el": -204.19573361799644,
        "E_en": -487.3652607501766,
        "E_kin": 200.15225557087115,
        "E_next": 0.0,
        "E_nn": 3.6712302669537533,
        "E_nuc": 3.6712302669537533,
        "E_tot": -200.52450335104268,
        "E_x": -4.084189960357476,
        "E_xc": -12.626363106806387,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 99.13585010499179,
        "E_eext": 0.0,
        "E_el": -203.0056279332109,
        "E_en": -483.7534637371984,
        "E_kin": 198.29097671844875,
        "E_next": 0.0,
        "E_nn": 3.6712302669537533,
        "E_nuc": 3.6712302669537533,
        "E_tot": -199.33439766625713,
        "E_x": -4.081125385868832,
        "E_xc": -12.597865633584203,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 99.29134382433227,
              "E_eext": 0.0,
              "E_el": -204.15247868407545,
              "E_en": -484.6851310819837,
              "E_kin": 197.89344999631203,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.4812484171217,
              "E_x": -4.072349812514885,
              "E_xc": -12.57979161022119,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.4812484171217,
            "energy_update": 1.1468507508645587,
            "mo_residual": 0.1999147694811673,
            "wall_time": 75.197924751
          },
          {
            "energy_terms": {
              "E_ee": 99.47375214003645,
              "E_eext": 0.0,
              "E_el": -204.1887358436575,
              "E_en": -486.082572546862,
              "E_kin": 199.09800193424655,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.51750557670374,
              "E_x": -4.077320291528657,
              "E_xc": -12.600597079549821,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.51750557670374,
            "energy_update": 0.03625715958204978,
            "mo_residual": 0.049862350398346356,
            "wall_time": 51.231921396
          },
          {
            "energy_terms": {
              "E_ee": 99.61127039725496,
              "E_eext": 0.0,
              "E_el": -204.19400128508545,
              "E_en": -486.80875460122945,
              "E_kin": 199.69981313053728,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.5227710181317,
              "E_x": -4.081245284722313,
              "E_xc": -12.615084926925912,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.5227710181317,
            "energy_update": 0.0052654414279516,
            "mo_residual": 0.02532105627401417,
            "wall_time": 43.37110803
          },
          {
            "energy_terms": {
              "E_ee": 99.69702756503551,
              "E_eext": 0.0,
              "E_el": -204.1952499163746,
              "E_en": -487.25849983249105,
              "E_kin": 200.07372791779935,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.52401964942084,
              "E_x": -4.083554208797127,
              "E_xc": -12.623951357921245,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.52401964942084,
            "energy_update": 0.0012486312891439866,
            "mo_residual": 0.021753595720740062,
            "wall_time": 44.779045499
          },
          {
            "energy_terms": {
              "E_ee": 99.74656979665201,
              "E_eext": 0.0,
              "E_el": -204.19566996270166,
              "E_en": -487.4892366531203,
              "E_kin": 200.26071233171425,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.5244396957479,
              "E_x": -4.084881218336067,
              "E_xc": -12.628834219611486,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.5244396957479,
            "energy_update": 0.00042004632706493794,
            "mo_residual": 0.015010492391263877,
            "wall_time": 43.931051106
          },
          {
            "energy_terms": {
              "E_ee": 99.73468883967439,
              "E_eext": 0.0,
              "E_el": -204.1957146647324,
              "E_en": -487.4089220397496,
              "E_kin": 200.19020969202484,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.52448439777865,
              "E_x": -4.084418860610008,
              "E_xc": -12.627272296072023,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.52448439777865,
            "energy_update": 4.4702030749022015e-05,
            "mo_residual": 0.005128068405696234,
            "wall_time": 46.576777246
          },
          {
            "energy_terms": {
              "E_ee": 99.72986103959737,
              "E_eext": 0.0,
              "E_el": -204.19572941720946,
              "E_en": -487.36889690961567,
              "E_kin": 200.15418408773078,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.5244991502557,
              "E_x": -4.084264084832556,
              "E_xc": -12.626613550089404,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.5244991502557,
            "energy_update": 1.4752477056845237e-05,
            "mo_residual": 0.002226830716654989,
            "wall_time": 46.932620265
          },
          {
            "energy_terms": {
              "E_ee": 99.7267668428324,
              "E_eext": 0.0,
              "E_el": -204.19573247868937,
              "E_en": -487.3547391280578,
              "E_kin": 200.14269078575592,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.5245022117356,
              "E_x": -4.084180120556377,
              "E_xc": -12.626270858663494,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.5245022117356,
            "energy_update": 3.061479901589337e-06,
            "mo_residual": 0.0016524150075751473,
            "wall_time": 44.537587295
          },
          {
            "energy_terms": {
              "E_ee": 99.72567756717332,
              "E_eext": 0.0,
              "E_el": -204.19573308058173,
              "E_en": -487.35445382267704,
              "E_kin": 200.14336619261434,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.52450281362798,
              "E_x": -4.084147911845052,
              "E_xc": -12.626175105847253,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.52450281362798,
            "energy_update": 6.018923670580989e-07,
            "mo_residual": 0.0009775540370515678,
            "wall_time": 46.127506225
          },
          {
            "energy_terms": {
              "E_ee": 99.72703539941824,
              "E_eext": 0.0,
              "E_el": -204.19573342454402,
              "E_en": -487.3621061930078,
              "E_kin": 200.1498369688892,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.52450315759026,
              "E_x": -4.084186495177731,
              "E_xc": -12.626313104665927,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.52450315759026,
            "energy_update": 3.4396228443256405e-07,
            "mo_residual": 0.0007181860672895946,
            "wall_time": 46.019640695
          },
          {
            "energy_terms": {
              "E_ee": 99.72772615822201,
              "E_eext": 0.0,
              "E_el": -204.19573359329038,
              "E_en": -487.36586651292,
              "E_kin": 200.15297246974575,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.52450332633663,
              "E_x": -4.084195172431364,
              "E_xc": -12.626370535906819,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.52450332633663,
            "energy_update": 1.6874636799002474e-07,
            "mo_residual": 0.00026288926274158563,
            "wall_time": 49.097310622
          },
          {
            "energy_terms": {
              "E_ee": 99.72782462847287,
              "E_eext": 0.0,
              "E_el": -204.19573361799644,
              "E_en": -487.3652607501766,
              "E_kin": 200.15225557087115,
              "E_next": 0.0,
              "E_nn": 3.6712302669537533,
              "E_nuc": 3.6712302669537533,
              "E_tot": -200.52450335104268,
              "E_x": -4.084189960357476,
              "E_xc": -12.626363106806387,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -200.52450335104268,
            "energy_update": 2.4706054091439e-08,
            "mo_residual": 8.338876921855644e-05,
            "wall_time": 54.844653385
          }
        ],
        "wall_time": 592.684981845
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
