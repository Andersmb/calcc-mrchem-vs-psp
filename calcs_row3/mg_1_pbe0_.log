Starting job 5688398 on c1-26,c2-35,c10-[7,52],c11-60 at Fri May 20 11:11:12 CEST 2022

Copying mg_1_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying mg_1_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688398      mg_1_pbe0_        180                                             00:11:08      0:0 
5688398.bat+      batch         24        1   00:00:02          0   00:00:02   00:11:08      0:0 
5688398.ext+     extern        180        5   00:00:00          3   00:00:00   00:11:09      0:0 
5688398.0      mrchem.x        180       30   00:10:49         29   00:20:56   00:10:56      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688398                                                                          
5688398.bat+     19276K          0     19276K        0              0          0 
5688398.ext+          0          3          0        0              3          0 
5688398.0      3580420K         29 2450909047        0             27          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688398                                                                                               
5688398.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688398.ext+        0.00M               3          0.00M            0                3              0 
5688398.0           7.32M              20          7.12M       63.13M                0         26.11M 

Job 5688398 completed at Fri May 20 11:22:21 CEST 2022
