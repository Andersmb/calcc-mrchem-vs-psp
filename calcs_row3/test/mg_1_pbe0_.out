                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Mg            :        0.000000        0.000000       -0.130155
    1       H            :        0.000000        0.000000        3.138504
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               7
 Beta electrons          :                                               6
 Total electrons         :                                              13
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000001379e+00
    1    1    a          :                              9.999999969172e-01
    2    1    a          :                              9.999999923585e-01
    3    1    a          :                              9.999999990279e-01
    4    1    a          :                              9.999999990279e-01
    5    1    a          :                              1.000000068819e+00
    6    1    a          :                              1.000000108462e+00
    7    1    b          :                              1.000000001379e+00
    8    1    b          :                              9.999999969172e-01
    9    1    b          :                              9.999999923585e-01
   10    1    b          :                              9.999999990279e-01
   11    1    b          :                              9.999999990279e-01
   12    1    b          :                              1.000000068819e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                198.290976718449
 E-N energy              :            (au)               -483.753463737198
 Coulomb energy          :            (au)                 99.135850104992
 Exchange energy         :            (au)                 -4.081125385869
 X-C energy              :            (au)                -12.597865633584
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  3.671230266954
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -203.005627933211
 Nuclear energy          :            (au)                  3.671230266954
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.993343976663e+02
                         :      (kcal/mol)             -1.250842220330e+05
                         :        (kJ/mol)             -5.233523853257e+05
                         :            (eV)             -5.424165048606e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -46.651507341239
    1    1    a          :            (au)                 -3.169560998448
    2    1    a          :            (au)                 -1.888277627512
    3    1    a          :            (au)                 -1.888277621708
    4    1    a          :            (au)                 -1.884209145917
    5    1    a          :            (au)                 -0.270216743075
    6    1    a          :            (au)                 -0.164005611494
    7    1    b          :            (au)                -46.649061324847
    8    1    b          :            (au)                 -3.164131914477
    9    1    b          :            (au)                 -1.884671612223
   10    1    b          :            (au)                 -1.884671336581
   11    1    b          :            (au)                 -1.878721535523
   12    1    b          :            (au)                 -0.231609759432
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -111.608922572476
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          3.605551e+00        -199.334397666257         -1.993344e+02
    1          1.999148e-01        -200.481248417122         -1.146851e+00
    2          4.986235e-02        -200.517505576704         -3.625716e-02
    3          2.532106e-02        -200.522771018132         -5.265441e-03
    4          2.175360e-02        -200.524019649421         -1.248631e-03
    5          1.501049e-02        -200.524439695748         -4.200463e-04
    6          5.128068e-03        -200.524484397779         -4.470203e-05
    7          2.226831e-03        -200.524499150256         -1.475248e-05
    8          1.652415e-03        -200.524502211736         -3.061480e-06
    9          9.775540e-04        -200.524502813628         -6.018924e-07
   10          7.181861e-04        -200.524503157590         -3.439623e-07
   11          2.628893e-04        -200.524503326337         -1.687464e-07
   12          8.338877e-05        -200.524503351043         -2.470605e-08
---------------------------------------------------------------------------
                      SCF converged in 12 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Mg            :        0.000000        0.000000       -0.130155
    1       H            :        0.000000        0.000000        3.138504
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                200.152255570871
 E-N energy              :            (au)               -487.365260750177
 Coulomb energy          :            (au)                 99.727824628473
 Exchange energy         :            (au)                 -4.084189960357
 X-C energy              :            (au)                -12.626363106806
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  3.671230266954
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -204.195733617996
 Nuclear energy          :            (au)                  3.671230266954
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.005245033510e+02
                         :      (kcal/mol)             -1.258310246193e+05
                         :        (kJ/mol)             -5.264770073489e+05
                         :            (eV)             -5.456549472645e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -46.925309297905
    1    1    a          :            (au)                 -3.161247197678
    2    1    a          :            (au)                 -1.884248460829
    3    1    a          :            (au)                 -1.884248459900
    4    1    a          :            (au)                 -1.880671865341
    5    1    a          :            (au)                 -0.302959949667
    6    1    a          :            (au)                 -0.181067576070
    7    1    b          :            (au)                -46.923297987696
    8    1    b          :            (au)                 -3.157390592508
    9    1    b          :            (au)                 -1.881833365985
   10    1    b          :            (au)                 -1.881833362487
   11    1    b          :            (au)                 -1.875351411928
   12    1    b          :            (au)                 -0.277366718708
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -112.216826246702
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000       -2.081903
 Magnitude               :            (au)                        2.081903
                         :         (Debye)                        5.291669
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.576641
 Magnitude               :            (au)                        1.576641
                         :         (Debye)                        4.007421
---------------------------------------------------------------------------
 Total vector            :       -0.000000       -0.000000       -0.505262
 Magnitude               :            (au)                        0.505262
                         :         (Debye)                        1.284249
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 10m 49s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
